app/docs/0.11.x/clustering.md

```diff
@@ -4,54 +4,317 @@ title: Clustering Reference
+    - [Purge a node's cache][6-3]
+
+[1]: #what-does-kong-cluster-do-and-doesnt-do
+[2]: #single-node-kong-clusters
+[3]: #multiple-nodes-kong-clusters
+[4]: #what-is-being-cached
+[5]: #how-to-configure-database-caching
+[5-1]: #1-db_update_frequency-default-5s
+[5-2]: #2-db_update_propagation-default-0s
+[5-3]: #3-db_cache_ttl-default-3600s
+[5-4]: #when-using-cassandra
+[6]: #interacting-with-the-cache-via-the-admin-api
+[6-1]: #inspect-a-cached-value
+[6-2]: #purge-a-cached-value
+[6-3]: #purge-a-nodes-cache
+
+### What does Kong cluster do and doesn't do?
+
+**Having a Kong cluster does not mean that your clients traffic will be
+load-balanced across your Kong nodes out of the box.** You still need a
+load-balancer in front of your Kong nodes to distribute your traffic. Instead,
+a Kong cluster means that those nodes will share the same configuration.
+
+For performance reasons, Kong avoids database connections when proxying
+requests from downstream clients, and caches the contents of your database in
+memory. The cached entities include APIs, Consumers, Plugins, Credentials,
+etc... Since those values are in memory, any change made via the Admin API of
+one of the nodes needs to be propagated to the other nodes.
+
+This document describes how those cached entities are being invalidated and how
+to configure your nodes for your use case, which lies somewhere between
+performance and consistency.
+
+[Back to TOC](#table-of-contents)
+
+### Single node Kong clusters
+
+A single Kong node connected to a database (Cassandra or PostgreSQL) creates a
+Kong cluster of one node. Any changes applied via the Admin API of this node
+will instantly take effect. Example:
+
+Consider a single Kong node `A`. If we delete a previously registered API:
+
+```bash
+$ curl -X DELETE http://127.0.0.1:8001/apis/test-api
+```
+
+Then any subsequent request to `A` would instantly return `404 Not Found`, as
+the node purged it from its local cache:
+
+```bash
+$ curl -i http://127.0.0.1:8000/test-api
+```
+
+[Back to TOC](#table-of-contents)
+
+### Multiple nodes Kong clusters
+
+In the continuity of the above example, other Kong nodes connected to the same
+database would not instantly be notified that the API was deleted by node `A`.
+While the API is **not** in the database anymore (it was deleted by node `A`),
+it is **still** in node `B`'s memory.
+
+All nodes perform a background job that is periodically ran to synchronize with
+any change that may have been triggered by another node. This job will run at a
+given interval, which you can configure via:
+
+* [db_update_frequency][db_update_frequency] (default: 5 seconds)
+
+Every `n` seconds, all running Kong nodes will poll the database for any
+update, and will purge their cache if necessary.
+
+Following the previous example in which we deleted an API from node `A`, then
+this change will be effective in node `B` on its next database poll, which
+might be up to `n` seconds later (it could be sooner too).
```

Comment:
"Following the previous example in which we deleted an API from node `A`, then
 this change will be effective in node `B` on its next database poll, which
 might be up to `n` seconds later (it could be sooner too)." -> "If we delete an API from node `A`,
 this change will not be effective in node `B` until node `B`s next database poll, which
 will occur up to `db_update_frequency` seconds later (though it could happen sooner)."